internal/fs/inode/file.go

```diff
@@ -414,6 +414,14 @@ func (f *FileInode) Attributes(
 		}
 	}
 
+	bwEnabled, err := f.shouldTriggerBufferedWrites()
+	if err != nil {
+		return
+	}
+	if bwEnabled {
+		attrs.Mtime = f.bwh.WriteFileInfo().Mtime
```

Comment:
Should we take size also from bwh?

Reply:
Thanks. Since i focussed only on mtime, missed the size. updated now